app/adapters/file.js

```diff
@@ -0,0 +1,12 @@
+import ApplicationAdapter from './application';
+
+export default class FileAdapter extends ApplicationAdapter {
+  deleteRecord(store, type, snapshot) {
+    console.warn('File deletion cannot be rolled back');
```

Comment:
This probably doesn't need to be put in the console. Should I move to a code comment?

Reply:
I think this is fine.